.gitignore

```diff
@@ -6,3 +6,5 @@ build
 coverage
 .php_cs.cache
 .phpunit.result.cache
+
+.idea/
```

Comment:
When you're done, make a commit removing the .idea folder, and removes it from .gitignore.